src/modules/nats/nats_mod.c

```diff
@@ -60,6 +60,8 @@ static param_export_t params[] = {
 static cmd_export_t cmds[] = {
 	{"nats_publish", (cmd_function)w_nats_publish_f,
 		  2, fixup_publish_get_value, fixup_publish_get_value_free, ANY_ROUTE},
```

Comment:
are these two lines still needed?

Reply:
I removed and rebased to the latest master, seems fine for me. Thanks.